charts/plum-recipe-backend/values.yaml

```diff
@@ -1,5 +1,6 @@
 java:
   applicationPort: 4550
+  readinessPath: '/health/readiness'
```

Comment:
we might want to make a breaking change to chart-java to change this? and put in the release notes it's for the newest spring boot and what to do,

thoughts: @hmcts/platform-engineering ?